### ISSUE TITLE
20,000 Leagues Under the Sea (1988)

### ISSUE THREAD

**Issue by parricc** [exoscoriae/eXoDOS#5362]
According to the manual, this game should have mouse control. However, after launching the game, DOSBox does not accept mouse input.

**Comment by SmilingSpectre** [comment]
It really talks about mouse, but I am not sure that PC supports it. Manual seems concerned all platforms in once - PC, Amstrad, Amiga and Atari - but is there mouse anywhere except Atari is a question.

Also, looking on a very bad colors, I suspect it's another (only lesser) case of color substitution from Amiga or something. :/

**Comment by Python-Exoproject** [comment]
I agree with Spectre, I dont think the mouse is for the PC version from the testing I did.

I also agree that this game suffers the bad graphics conversion shared with Emmanuel and the Brad Stallion games. Kudgel fixed the Brad Stallion games and started on Emmanuel which shows fixing it is possible so ill mark the ticket as low priority in the hope someone will complete Kudgels work one day.

Linking the Emmanuel ticket as well: #2144
